TFS.py

```diff
@@ -242,7 +249,6 @@ async def update(self, ctx, *, user: discord.Member = None):
             await ctx.send("Updated role to Member (Inactive) for " + user.name + ".")
 
     @commands.admin_or_permissions(manage_roles=True)
-    @commands.command()
```

Comment:
This is the command to update roles for all users, is there an alternative to this somewhere?

Reply:
`@commands.admin_or_permissions(manage_roles=True)` is restricted but `@commands.command()` is not. 
In the previous case `@commands.command()` override the restricted version so the command was not actually restricted.

Reply:
I might be wrong about that

Reply:
_Oh._ The way I saw it in some example somewhere was exactly like that. I assume you've checked that the command works like this, so fair enough.